android/KMAPro/kMAPro/src/main/java/com/tavultesoft/kmapro/MainActivity.java

```diff
@@ -848,7 +864,9 @@ public void onPackageInstalled(List<Map<String, String>> keyboardsInstalled) {
 
   @Override
   public void onLexicalModelInstalled(List<Map<String, String>> lexicalModelsInstalled) {
-    String langId = KMManager.getCurrentKeyboardInfo(this).get(KMManager.KMKey_LanguageID);
+    String langId = (KMManager.getCurrentKeyboardInfo(this) != null) ?
```

Comment:
Is this logic correct? If there is no current keyboard we want to register any models that match the default language?

Reply:
I ran into this timing issue where the app had installed the default model, but KMManager hadn't finished setting the current keyboard (to SIL EuroLatin).

Normally, there's always a current keyboard.